config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  match "/404", :to => "errors#not_found", :via => :all
+  match "/500", :to => "errors#internal_server_error", :via => :all
+
```

Comment:
The problem with 500 pages defined using controllers is that if the whole app breaks it can't even be rendered. That's why there's 404.html and 500.html in /public by default. We're likely to have an nginx server in front of the app anyway, which will serve its own static 500 page if the application is down.